torchci/pages/api/classifier/rules.ts

```diff
@@ -179,6 +179,16 @@ export default async function handler(
         pattern: r`undefined reference to .*`,
         priority: 200,
       },
+      {
```

Comment:
i think these should be added to ruleset.toml? suo changed it recent and idk if this is even used anymore